src/lib/selector/ProductSelector.ts

```diff
@@ -127,14 +127,14 @@ export class ProductSelector {
     }
 
     public async getResourceJunctionObjects(connection: Connection, productIds: Array<String>) {
-        //todo: add the productResource to schema class
       try{
-        const productResourcequery = "SELECT enxCPQ__Resource__r.enxCPQ__TECH_External_Id__c, enxCPQ__Criteria__c, enxCPQ__Assignment_Criteria__c, enxCPQ__Applicable_Solution_Variants__c, enxCPQ__Product__r.enxCPQ__TECH_External_Id__c, CurrencyIsoCode, enxCPQ__Product__c, enxCPQ__TECH_External_Id__c, enxCPQ__Resource__c, enxCPQ__Resource_Location_Type__c \
+        const queryLabel = 'productRes';
```

Comment:
Ten label będzie potem się wyświetlał użytkownikom końcowym i z doświadczenia - dobrze jest jak jest dosłowny, a nie skrótowy, bo potem wszyscy pytają co im się wysypało w ProductRes

Reply:
Tutaj jest problem z tym, że w pliku konfiguracyjnym nazwaliśmy obiekty Product(RT Resource) jako productResources, dlatego chciałem dać skrót, żeby nie było różnicy tylko w jednej literce. A nie wystarczy tylko jako label do executeQuery zostanie po staremu ?

Reply:
Pewnie, to starczy. Po prostu wielu ludzi w historii dostawało linijki, których nie rozumieli :)